dedalus/core/distributor.py

```diff
@@ -395,6 +396,7 @@ def decrement_single(self, field):
         cdata = field.data
         # Transform non-constant bases with local data
         if (basis is not None) and np.prod(gdata.shape):
+            (gdata.shape, cdata.shape, field.domain.bases[0].shape, self.layout0.grid_space)
```

Comment:
What is this doing?

Reply:
Oops looks like its just a leftover `print` fragment.

Reply:
Fixed